WordPress/Classes/ViewRelated/Post/AztecPostViewController.swift

```diff
@@ -1933,6 +1943,45 @@ extension AztecPostViewController {
         static let acceptTitle  = NSLocalizedString("Yes", comment: "Accept Action")
         static let cancelTitle  = NSLocalizedString("Not Now", comment: "Nicer dialog answer for \"No\".")
     }
+
+    struct ToolBarItemsAccessibility {
+        static let mediaLabel = NSLocalizedString("Insert media", comment: "Accessibility label for insert media button on formatting toolbar.")
+        static let mediaIdentifier = "format_toolbar_insert_media"
```

Comment:
Suggestion (no need to make any changes!):

```
enum ToolBarItems {
   case .header
   case .bold
   case .italic
   // ...

    var accessibilityLabel: String {
       switch self {
         case .bold: return ""
         ///
       }

    var accessibilityIdentifier: String {
       switch self {
         case .bold: return ""
         ///
       }
    }

    var image: UIImage {  //.. }
    var identifier: String { // }
}
```

Benefits there would be:
- Whenever a new Item is added, the compiler shows a warning for the two new properties that must be added
- The makeToolbarButton's signature could look as follows:

```
  func makeToolbarButton(for item: ToolBarItems) -> FormatBarItem {
       let button = FormatBarItem(image: item.image, identifier: item.identifier)
       button.accessibilityLabel = item.accessibilityLabel
       button.accessibilityIdentifier = item.accessibilityIdentifier
       return button
   }
```

**No need to change anything**, just a potential way to simplify a couple calls here and there!!